extensions/wikia/CommunityPage/models/CommunityPageSpecialInsightsModel.class.php

```diff
@@ -144,12 +134,12 @@ private function getSortingParam( $sortingType ) {
 		return [];
 	}
 
-	private function getArticleContributeText( $metadata ) {
+	private function getArticleContributeText( $metadata, $type ) {
 		if ( !empty( $metadata['wantedBy'] ) ) {
 			return wfMessage( 'communitypage-page-list-create' )->text();
 		}
 
-		return wfMessage( 'communitypage-page-list-edit' )->text();
+		return wfMessage( 'communitypage-' . $type . '-edittext' )->text();
```

Comment:
Please add possible message keys in a comment above, per https://www.mediawiki.org/wiki/Localisation#Using_messages